editor/components/block-list/insertion-point.js

```diff
@@ -26,18 +26,21 @@ class BlockInsertionPoint extends Component {
 			<div className="editor-block-list__insertion-point">
 				{ showInsertionPoint && <div className="editor-block-list__insertion-point-indicator" /> }
 				{ showInserter && (
-					<button
-						className="editor-block-list__insertion-point-inserter"
-						onClick={ this.onClick }
-						aria-label={ __( 'Insert block' ) }
-					/>
+					<div className="editor-block-list__insertion-point-inserter">
+						<IconButton
+							icon="insert"
+							className="editor-block-list__insertion-point-button"
+							onClick={ this.onClick }
+							aria-label={ __( 'Insert block' ) }
```

Comment:
`aria-label`  doesn't work on the IconButton component, so this button is not labeled at all and announced just as "button". IconButton needs a `label` prop. That would also produce a tooltip, which is necessary to make the visually expose the name of the control.

Most importantly, the inserter doesn't show up on focus, when tabbing with the keyboard.